views/common/content_class.view.php

```diff
@@ -14,8 +14,16 @@
         <?php
         }
         break;
-
+        
+    case 'Novius\OnlineMediaFiles\Model_Media':
+        if (!empty($value)) {
```

Comment:
this `!empty` is not neccessary, $value _is_ an instance of Model_Media when you get here